### ISSUE TITLE
This is now broken on my strip. Christmas update worked fine.

### ISSUE THREAD

**Issue by omarkhwj** [not-matt/audio-reactive-led-strip#5]
All visualizations are buggy/random noise looking. Scroll and Energy will crash the program as soon as I click them. Is there anyway I can give you more info for debugging.

**Comment by not-matt** [comment]
Yes that's fine, this is a large change in the program regarding flexibility. Email me as much appropriate info and I'll get on it.

**Comment by not-matt** [comment]
For now though, keep using the Christmas update - this is work in progress. 👍
